### PR TITLE
#776. add phrase button in analysis - look change

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js [hunk 1]
```diff
@@ -233,8 +233,10 @@ var initAnalysisJs = function() {
      */
     function initAddNewSearchPhraseButton() {
         
-        $('#addPhraseButton').click(function(){
-        
+        $('#addPhraseButton').click(function(event){
+              
+              event.preventDefault();
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js]: Can you add a small comment why preventDefault here?

File: saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/newLayout.scss [hunk 2]
```diff
@@ -841,7 +845,7 @@ a, .hint {
     
     .navigation {
     
-        input[type="text"] {
+        input[type="text"], button.input-button {
             padding-left: 35px;
             
         }
```
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/newLayout.scss]: I think the input-button should be similar but not identical to phrase inputs. I would make it a little more grayish. E.g. color of the border could be aaa, color of the span background 777. Additionaly, i would change 'Dodaj hasło' to '+ Dodaj hasło'.

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js (hunk 1 above)
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js]: ok

File: saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/newLayout.scss (hunk 2 above)
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/stylesheet/scss/newLayout.scss]: :+1:

**Pull request closed without merging**